Formula/zenio.rb

```diff
@@ -0,0 +1,15 @@
+class Zenio < Formula
+  homepage "http://dev.zenio.co"
+  url "http://dev.zenio.co/cli/zenio-0.9.0-SNAPSHOT.tar"
```

Comment:
Homebrew core only ships stable, released versions of open source software.

Reply:
fixed.